wsagent/che-core-workspace-activity-server/src/main/java/org/eclipse/che/workspace/activity/WorkspaceActivityNotifier.java

```diff
@@ -75,10 +75,17 @@ private void scheduleActivityNotification() {
   }
 
   private void notifyActivity() {
-    try {
-      httpJsonRequestFactory.fromUrl(apiEndpoint + "/activity/" + wsId).usePutMethod().request();
-    } catch (Exception e) {
-      LOG.error("Cannot notify master about workspace " + wsId + " activity", e);
-    }
+    new Thread(
```

Comment:
We need new Thread only when `onActivity` method is called but not when `scheduleActivityNotification`. Then maybe it would be better to have two methods: `notifyActivity` and `notifyActivityAsync`?